tests/dragonfly/replication_test.py

```diff
@@ -1403,20 +1403,6 @@ async def test_tls_replication(
     db_size = await c_replica.execute_command("DBSIZE")
     assert 100 == db_size
 
-    # 4. Kill master, spin it up and see if replica reconnects
```

Comment:
So we want to check reconnect with tls here.
Instead of removing this check lets think of another way to make the reconnect work like not using stop to break the connection but use the proxy instead

Reply:
what do you mean by proxy? do we have such a thing?

Reply:
take a look into test_network_disconnect test

Reply:
Nice! Done.